askama_shared/src/path.rs

```diff
@@ -32,34 +32,23 @@ pub fn get_template_source(tpl_path: &Path) -> String {
 }
 
 pub fn find_template_from_path(path: &str, start_at: Option<&Path>) -> PathBuf {
-    let root = template_dir();
+    let dirs = template_dirs();
     if let Some(rel) = start_at {
-        let mut fs_rel_path = root.clone();
-        fs_rel_path.push(rel);
-        fs_rel_path = fs_rel_path.with_file_name(path);
+        let root = search_template_in_dirs(&dirs, rel).unwrap();
```

Comment:
Not sure `unwrap()` makes sense here?

Reply:
I think since `start_at` is defined this is a path that already existed previously.

Reply:
This should also `expect()` instead of `unwrap()`.